### PR TITLE
Dockerize Clearance UI

### PR DESCRIPTION
**Pull request by mmurto**
Details in commits.

### CODE REVIEW

**Comment by mmurto** [comment]
I'll actually add the publishing action to this PR as well.

File: .github/workflows/playwright.yml [hunk 1]
```diff
@@ -10,29 +10,28 @@ on:
...
-    NEXTAUTH_URL: http://localhost:3000
-    NEXTAUTH_SECRET: ${{ secrets.NEXTAUTH_SECRET }}
-    NEXT_PUBLIC_API_URL: http://localhost:3001/api/
-
 jobs:
     test:
         timeout-minutes: 60
         runs-on: ubuntu-latest
         environment: test-environment
+        env:
```
Comment (sschuberth) [inline .github/workflows/playwright.yml:18]: Commit message nit (optional):
- "VSCode provided" could be "VSCode-provided" or "autocomplete for secrets
doesn't work in VSCode"

File: UI.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+RUN adduser --system --uid 1001 nextjs
+USER nextjs
+
+# Automatically leverage output traces to reduce image size
+# https://nextjs.org/docs/advanced-features/output-file-tracing
+COPY --from=installer --chown=nextjs:nodejs /app/apps/clearance_ui/.next/standalone ./
+COPY --from=installer --chown=nextjs:nodejs /app/apps/clearance_ui/.next/static ./apps/clearance_ui/.next/static
+COPY --from=installer --chown=nextjs:nodejs /app/apps/clearance_ui/public ./apps/clearance_ui/public
+
+# server.js is created by next build from the standalone output
```
Comment (sschuberth) [inline UI.Dockerfile:41]: Nit: Maybe "next" -> "Next.js", as otherwise it reads like "by the next build".

File: .github/workflows/docker.yml [hunk 3]
```diff
@@ -89,3 +89,28 @@ jobs:
                   labels: ${{ steps.meta-api.outputs.labels }}
                   cache-from: type=local,src=/tmp/.buildx-cache
                   cache-to: type=local,dest=/tmp/.buildx-cache
+
+            - name: Docker metadata for Clearance UI
```
Comment (sschuberth) [inline .github/workflows/docker.yml:93]: Commit message: Shouldn't "compatible by our deployment" say "compatible *with* our deployment"?

Also, if we want to follow impersonal wording here as well, we should replace "our" with e.g. "Double Open's".

**Review by sschuberth — APPROVED** [review]
Only optional wording stuff. Still approving in case you want to move forward quickly.

**Pull request merged**